fiber/chain/weights.py

```diff
@@ -217,3 +217,12 @@ def set_node_weights(
 
     substrate.close()
     return success
+
+
+def get_node_weights(
```

Comment:
if this is get the weights set by a generic node, just node_id instead of validator_node_id? 

get_weights_set_by_node is a clearer name IMO (as opposed to the weights set FOR a node, this func name could be either)